### ISSUE TITLE
Smile character  - code: $263A (Left Alt+1)

### ISSUE THREAD

**Issue by fair2003** [CatharsisFonts/Cormorant#51]
Could the Cormorant Garamond font be enriched with a smile character (code: $263A)? I created a smile character based on capital letter O – see enclosed file. In my opinion, it looks very nice. You can use it.
Ridero.eu uses Cormorant Garamond font and I need this character very much :).

**Comment by fair2003** [comment]
[CormorantGaramond-Regular-with-smile.zip](https://github.com/CatharsisFonts/Cormorant/files/4209730/CormorantGaramond-Regular-with-smile.zip)

**Comment by CatharsisFonts** [comment]
Good idea! I drew my own to match the Cormorant style, though:
<img width="963" alt="Screenshot 2020-02-16 at 15 42 13" src="https://user-images.githubusercontent.com/13754106/74607071-c3b69380-50d5-11ea-8485-5a5a827346db.png">
It's in the newest commit.

**Comment by CatharsisFonts** [comment]
BTW, I'm assuming you're using Cormorant for titles? I don't recommend it for text; it's too delicate for small sizes.
I do recommend [Ysabeau](https://github.com/CatharsisFonts/Ysabeau) for text, though, at least up to Bold weight. The blacker weights beyond that are not finished yet.

**Comment by fair2003** [comment]
Thank you very much !! It looks great but maybe you could gently increase the distance between the eyes :).
I agree with you that Cormorant is a very thin, delicate font. Unfortunately in Ridero, the user cannot freely choose the fonts. There are only a few to choose from. In my opinion, the most beautiful among them is Cormorant Garamond. I will choose size 12, so it shouldn't be that bad.
Can you make the new version of font appear on fonts.google.com? That's where people download fonts most often and probably Ridero too :).
Best regards, Andy

**Comment by CatharsisFonts** [comment]
I did a bit more tuning on the face, but couldn't move the eyes more than a few units before it looked like too much to me.

I'm flattered that you like Cormorant Garamond best among the choices, but beauty is not the only criterion in choosing a book font. A size of 12 pt might seem like much for text, but it's small for the purposes of a display typeface like Cormorant. 

Google Fonts updates their font library only rarely; I don't know when the next time is due. Maybe you can convince Rideró to grab the newest version of Cormorant off GitHub.

**Comment by fair2003** [comment]
Thank you very much for your help.
I agree with you that Cormoran Garamond is a little too delicate for book body text. In my text editor I use Garamond Monotype, which is a bit fatter. I will think about all this yet. I made test printouts with all the fonts available in Ridero and tomorrow I will compare them in daylight.
I already wrote to Ridero about new version of Cormoran font on GitHub, but I'm not sure what they will do :).
Best regards from Poland :).
